pac4j-cas/src/main/java/org/pac4j/cas/client/rest/AbstractCasRestClient.java

```diff
@@ -70,7 +71,7 @@ public void destroyTicketGrantingTicket(final WebContext context, final HttpTGTP
             final URL deleteURL = new URL(endpointURL, endpointURL.getPath() + "/" + profile.getTicketGrantingTicketId());
             connection = HttpUtils.openDeleteConnection(deleteURL);
             final int responseCode = connection.getResponseCode();
-            if (responseCode != HttpStatus.SC_OK) {
+            if (responseCode != HTTP_STATUS_OK) {
```

Comment:
You already have this constant here: https://github.com/pac4j/pac4j/blob/master/pac4j-core/src/main/java/org/pac4j/core/context/HttpConstants.java